Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Db.ts

```diff
@@ -37,19 +38,18 @@ export async function checkExistsAndCreateTables(client: Client) {
  * Create a connection pool, check that the database and table structure is in place
  * or create it if not and then return the pool
  */
-export async function createConnectionPoolAndReturnClient(): Promise<Client> {
-  let client: Client;
```

Comment:
This was actually two different types, sometimes Client, sometimes PoolClient. They have similar signatures, which is why this worked in practice.